iridium-client-nextjs/src/lib/services/accessToken.service.ts

```diff
@@ -0,0 +1,25 @@
+import { CookieService } from './cookie.service';
+import { OauthConstants } from './oauth-constants';
+import { useEffect } from 'react';
```

Comment:
This is not being used. I'd personally remove it.

Also, useEffect can only be used in a React function component. This is a component that returns some jsx. Typically with a file extension .tsx. A good rule of thumb is a .ts file should not be using hooks that have the word "use"